menyer/kackerlackan/stylesheet.css

```diff
@@ -0,0 +1,88 @@
+/*För bodyn*/
+body {
+    background-image: url(assets/background.jpg);
+    background-size: cover;
+    margin: 3% 20% 5% 20%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+
+}
+
+/*Headings position*/
+h1, h2 {
+    text-align: center;
+    font-weight: 900;
+}
+
+/*för sectionerna, samt header och footer*/
+#forratter, #huvudratter, #efterratter, footer, header {
```

Comment:
Important: if have many selectors stacked like this ask yourself if you should perhaps change the selector to something they have all in common. If there is nothing unique about #foratt, #huvudratt,.. etc then don't use it. 

I would change this to a class and then apply it where it is appropriate.